lib/data/database.ts

```diff
@@ -33,21 +34,21 @@ export default class DatabaseService extends Service {
   }
 
   async query(modelType: string, query: any, options?: any): Promise<Model[]> {
-    debug(`${ modelType } query: ${ query }`);
     assert(query != null, `You must pass a query to Model.query(conditions)`);
     let adapter = this.lookupAdapter(modelType);
     let result = await adapter.query(modelType, query, options);
+    debug(`${ modelType } query: found ${result.length} records: ${ util.inspect(query) }`);
```

Comment:
Here and below, I'd rather not wait till after the adapter to log this stuff out - if the adapter is buggy, then knowing the query _before_ it gets handed off to the adapter (which could throw, or never return). Maybe split these into two debug statements, one before, one after?

Reply:
Sounds good